src/sum_terms.py

```diff
@@ -0,0 +1,14 @@
+"""Implementation of Sum of nth Term Kata on Code Wars."""
+from __future__ import division
```

Comment:
python 2 wanted to return an int from the division which was not what I wanted. Instead of hacking it I decided to make it fully compatible import division :) It's in the docs!